### PR TITLE
Story 34 and 35

### PR DESCRIPTION
**Pull request by RyN21**
Created relationship between merchant and user_merchant
Add story 34 - 35
Worked on merchant dashboard show page

### CODE REVIEW

File: app/controllers/merchant/dashboard_controller.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 class Merchant::DashboardController < Merchant::BaseMerchantController
-  def index
+  def show
```
Comment (sciencefixion) [inline app/controllers/merchant/dashboard_controller.rb:2]: Good work with all this controller madness!

File: config/routes.rb [hunk 2]
```diff
@@ -55,6 +55,7 @@
 
   namespace :merchant do
     get "/dashboard", to: "dashboard#index"
+    get "/merchant", to: "dashboard#index"
```
Comment (jKwibe) [inline config/routes.rb]: How come both routes are referring to the dashboard index.

File: app/models/merchant.rb [hunk 3]
```diff
@@ -1,6 +1,7 @@
 class Merchant <ApplicationRecord
   has_many :items, dependent: :destroy
   has_many :item_orders, through: :items
+  has_many :users
```
Comment (jKwibe) [inline app/models/merchant.rb:4]: Nice

File: app/models/merchant.rb [hunk 4]
```diff
@@ -26,4 +26,15 @@ def distinct_cities
     item_orders.distinct.joins(:order).pluck(:city)
   end
 
+  def pending_orders
+    Order.joins(:item_orders).group(:id).where('orders.status = 0')
```
Comment (jKwibe) [inline app/models/merchant.rb:30]: Nice use of joins.

**Pull request merged**